src/installer/pkg/projects/nativeaot-packages.proj

```diff
@@ -1,9 +1,9 @@
 <Project>
   <Import Sdk="Microsoft.Build.Traversal" Project="Sdk.props" />
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(BuildNativeAOTRuntimePack)' != 'true'">
```

Comment:
I'm not exactly sure what the intention of that property was initially. Suppress both the meta package and the RID specific package? If yes, then the change looks good.

Reply:
Yes, we don't need these packages when building the runtime packs.